pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -320,7 +322,7 @@ stdenv.mkDerivation {
       rm $out/nix-support/setup-hook.tmp
 
       # some linkers on some platforms don't support specific -z flags
-      hardening_unsupported_flags=""
+      export hardening_unsupported_flags=""
```

Comment:
Without this `export`, `substitute` failed. but more more broadly I'm surprised it previously worked at all: `substituteAll` only crawls through the output of `env`, not `set`, yet it's important this variable be substituted. Maybe it was only on some branches exported?

CC @fpletz

Reply:
I checked in both builds and `@hardening_unsupported_flags@` was substituted.